### DISCUSSION TITLE
Icons only for Social networks

### DISCUSSION THREAD

**Discussion by timscha** [JulianPrieber/littlelink-custom#641] [Questions] [answered]
Hi! I just setup my Linkstack. Now I trying to create my links. How can I add Icons as links only as seen in the screenshot of themes?
Like the Instagram icon, etc... on top without any kind of text?
I wasn't able to find an option for this to enable this.

Thanks in advance
Tim
![CleanShot 2023-12-08 at 15 18 53](https://github.com/LinkStackOrg/LinkStack/assets/4650393/42fc5ba4-020d-40e9-9ffb-36cb66e60a64)

**Comment by JulianPrieber** [comment, marked as answer]
On the "Links" tab on your dashboard, there is the section "Page Icons". There you can add all the icons you need.

**Reply by timscha** [reply]
Thanks @JulianPrieber ! I haven‘t seen it as i only scrolled until the preview part. :)
